docs/user-guide/cli.md

```diff
@@ -124,6 +124,10 @@ A filename to assign the input. [More info](options.md#codefilename).
 
 Accept stdin input even if it is empty.
 
+### `--validate, --no-validate`
+
+Force enable/disable the validation of the rules' options.
```

Comment:
[suggestion] Let's add a link to `options.md`:

```suggestion
Force enable/disable the validation of the rules' options. [More info](options.md#validate).
```

Reply:
That would have been misleading.
i.e. it was the same exact description

Ill add the example first.